### PR TITLE
RUSSIAN SPEAK for BEEFMEN

### PR DESCRIPTION
**Pull request by SmoSmoSmoSmok**
## About The Pull Request

Beefmen can speak russian

## Why It's Good For The Game

Because they can SPEAK RUSSIAN. 
## Changelog

:cl:
russian speak for beefmen
/:cl:

### CODE REVIEW

File: fulp_modules/main_features/species/beefmen/mob/beefman.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	default_color = "#e73f4e"
 	species_traits = list(NOEYESPRITES, NO_UNDERWEAR, DYNCOLORS, AGENDER, HAS_FLESH, HAS_BONE)
 	mutant_bodyparts = list("beefcolor" = "Medium Rare", "beefmouth" = "Smile1", "beefeyes" = "Olives")
+	mutanttongue= /obj/item/organ/tongue/beefman_tongue
```
Comment (JohnFulpWillard) [inline fulp_modules/main_features/species/beefmen/mob/beefman.dm:15]:
```suggestion
	mutanttongue = /obj/item/organ/tongue/beefman_tongue
```

File: fulp_modules/main_features/species/beefmen/mob/beefman.dm [hunk 2]
```diff
@@ -722,3 +723,34 @@
...
+		/datum/language/beachbum,
+		/datum/language/aphasia,
+		/datum/language/piratespeak,
+		/datum/language/moffic,
+		/datum/language/sylvan,
+		/datum/language/shadowtongue,
+		/datum/language/terrum,
+		/datum/language/nekomimetic,
+		/datum/language/russian,
+		/datum/language/buzzwords
```
Comment (JohnFulpWillard) [inline fulp_modules/main_features/species/beefmen/mob/beefman.dm:751]:
```suggestion
		/datum/language/buzzwords,
```

File: fulp_modules/main_features/species/beefmen/mob/beefman.dm [hunk 3]
```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
+	name = "tongue"
```
Comment (JohnFulpWillard) [inline fulp_modules/main_features/species/beefmen/mob/beefman.dm:728]:
```suggestion
	name = "meaty tongue"
```

**Review by JohnFulpWillard — APPROVED** [review]

File: fulp_modules/main_features/species/beefmen/mob/beefman.dm (hunk 3 above)
Comment (SmoSmoSmoSmok) [reply fulp_modules/main_features/species/beefmen/mob/beefman.dm:728]: OK ASSHOLE how do i change the code and edit the PR look here and listen carefully this is my second time using github ok?

**Comment by SmoSmoSmoSmok** [comment]
did I fix it

File: fulp_modules/main_features/species/beefmen/mob/beefman.dm [hunk 4]
```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
+	name = "Meaty sausage tongue"
```
Comment (JohnFulpWillard) [inline fulp_modules/main_features/species/beefmen/mob/beefman.dm:728]:
```suggestion
	name = meaty sausage tongue"
```

Comment (SmoSmoSmoSmok) [reply fulp_modules/main_features/species/beefmen/mob/beefman.dm:728]: are U just TRYING to be an ASSHOLE mane becoz if u are then ur SUcceeding

File: fulp_modules/main_features/species/beefmen/mob/beefman.dm [hunk 5]
```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
+	name = "meaty sausage tongue"
+	desc = "A meaty and thick muscle typically found in Beefmen"
```
Comment (JohnFulpWillard) [inline fulp_modules/main_features/species/beefmen/mob/beefman.dm:729]:
```suggestion
	desc = "A meaty and thick muscle typically found in Beefmen."
```

last one i swear

Comment (SmoSmoSmoSmok) [reply fulp_modules/main_features/species/beefmen/mob/beefman.dm:729]: OK man but don’t think I don’t KNOW what’s going on here. CLEARLY ur a tyrant and U just don’t want Beefmen to have freedom of SPEECH. guess what????? I’m GOING to KEEP fighting U and this day WILL go dowN in history as The DAY I deFeated John Willard in the Name OF BEEFMEN civil LIBERTIES

Comment (JohnFulpWillard) [reply fulp_modules/main_features/species/beefmen/mob/beefman.dm:729]: i have your ip address

Comment (SmoSmoSmoSmok) [reply fulp_modules/main_features/species/beefmen/mob/beefman.dm:729]: hey man I was JOKING calm down OK? Listen when I GO home I’ll add ur GODDAMNED full stop

**Review by JohnFulpWillard — APPROVED** [review]

**Comment by JohnFulpWillard** [comment]
You'll have to fix the merge conflict imsosorry

**Comment by SmoSmoSmoSmok** [comment]
MAAAAAAAAAAAAAAAN SHIIIIIIIIIIIIIIIIIIEEEEEEEEEEET

**Comment by SmoSmoSmoSmok** [comment]
P

**Pull request closed without merging**